openy.profile

```diff
@@ -635,6 +707,10 @@ function openy_install_tasks_alter(&$tasks, &$install_state) {
     isset($tasks["openy_third_party_services"])) {
       unset($tasks["openy_third_party_services"]);
   }
+  // Remove Solr configure installation task for non search_api sorl service.
+  if (!\Drupal::state()->get('openy_show_solr_config') && isset($tasks["openy_solr_search"])) {
```

Comment:
You don't need to check it with `isset($tasks["openy_solr_search"])` here, because `unset($tasks["openy_solr_search"]);` will work fine even if `openy_solr_search` key doesn't exists in the array.
```
if (!\Drupal::state()->get('openy_show_solr_config')) {
```
Is enough

Reply:
Fixed